app/src/main/kotlin/no/nav/tiltakspenger/vedtak/clients/meldekort/MeldekortApiHttpClient.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.tiltakspenger.vedtak.clients.meldekort
+
+import arrow.core.Either
+import arrow.core.left
+import io.ktor.client.request.bearerAuth
```

Comment:
Tanken er å slette den ktor-klienten når vi får tid. Trenger ikke et eget bibliotek eller 10 for dette. Følger innebygd med i java 8/11.